### ISSUE TITLE
Parameter modification

### ISSUE THREAD

**Issue by yjydiancom** [dcaffo98/path-planning-cnn#4]
I would still like to refer to your model, but I have added all the data sets and modified one parameter(BATCH_SIZE = 160), but the training results are still not good(New min:5.533652072689051e-05). I would like to ask if there are any other parameters that need to be modified in the code you have shown?

**Comment by yjydiancom** [comment]
I ran it out and the loss function was not changed to MSEloss before.

**Comment by yjydiancom** [comment]
I feel that the results of my training are still a little wrong.
![image](https://github.com/dcaffo98/path-planning-cnn/assets/91858119/462898ab-45b4-425b-b6b1-7d6dda3e56f3)
For the same input，The image above was predicted using the checkpoints file you gave, and the image below was predicted by me using the results of my own training:
![image](https://github.com/dcaffo98/path-planning-cnn/assets/91858119/f7c6741c-3e4a-4ae7-b4af-b78ee35eac00)
I trained 25epoch with a minimum loss as shown below：
[Epoch 19]
        avg_loss: 2.7239708235624283e-05
        validation avg_loss: 3.0187028136300694e-05
        New min: 3.0187028136300694e-05 (previous was 3.0434074116445335e-05)
Can you help me see if there are any other parameters or others that have not been modified?It must be very disturbing to you, but I still hope you can help me, thank you!

**Comment by yjydiancom** [comment]
Occasionally there are such, and the rest is not much different.
